specification/azsadmin/resource-manager/infrastructureinsights/Microsoft.InfrastructureInsights.Admin/stable/2016-05-01/Alert.json

```diff
@@ -42,6 +45,9 @@
                         "schema": {
                             "$ref": "#/definitions/AlertList"
                         }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
                     }
```

Comment:
Unsuccessful response codes should not be included

Reply:
The team prefers not to throw an error on 404 and instead return null.

Reply:
I see. I highly recommend not setting it like this: other swaggers / SDKs do not follow this pattern, and it can be confusing for users when a SDK behaves in a different way.

---
In reply to: [181145822](https://github.com/Azure/azure-rest-api-specs/pull/2812#discussion_r181145822) [](ancestors = 181145822)

Reply:
I will bring this up with the PM who is leading the admin spec/SDK/PS push.

Reply:
What did you conclude there?